### PR TITLE
feat: add NSA Hardening Rules

### PR DESCRIPTION
**Pull request by myishay**

### CODE REVIEW

**Review by TzlilSwimmer123 — DISMISSED** [review]
Look great

File: pkg/policy/defaultRules.yaml [hunk 1]
```diff
@@ -1170,3 +1170,347 @@ rules:
                       required:
                         - retryPolicy
                         - expression
+  - id: 45
+    name: Ensure each container has a read-only root filesystem
+    uniqueName: CONTAINERS_INCORRECT_READONLYROOTFILESYSTEM_VALUE
+    enabledByDefault: false
+    documentationUrl: "https://hub.datree.io/"
```
Comment (royhadad) [inline pkg/policy/defaultRules.yaml]: missing documentationUrl

**Review by royhadad — CHANGES_REQUESTED** [review]

File: pkg/policy/defaultRules.yaml (hunk 1 above)
Comment (myishay) [reply pkg/policy/defaultRules.yaml]: Good catch! will add it now

**Comment by hadar-co** [comment]
https://github.com/datreeio/datree/pull/598/files

File: pkg/policy/defaultRules.yaml (hunk 1 above)
Comment (TzlilSwimmer123) [reply pkg/policy/defaultRules.yaml]: It's good to add it to defaultRules_test.go!

**Comment by myishay** [comment]
> https://github.com/datreeio/datree/pull/598/files

added them now

File: pkg/policy/defaultRules.yaml (hunk 1 above)
Comment (myishay) [reply pkg/policy/defaultRules.yaml]: @TzlilSwimmer123 added it now

**Review by royhadad — APPROVED** [review]

**Pull request merged**